### DISCUSSION TITLE
2.3.50 Grid Errors

### DISCUSSION THREAD

**Discussion by stondino00** [Security-Onion-Solutions/securityonion#4035] [Unsupported Versions] [answered]
When I run this from the master, all nodes show 100% green statuses of ok for all services running.

sudo salt \* so.status

When I go to the GRID section after upgrading to 2.3.50, I'm seeing this.

![image](https://user-images.githubusercontent.com/54154986/116453149-25b77080-a824-11eb-823d-f72de0624fb6.png)

[sohybridad@SO-Manager ~]$ sudo salt \* so.status

so-farmersville_heavynode:

    Checking Docker status

        Docker ---------------------------- [ OK ]

    Checking container statuses

        so-curator ------------------------ [ OK ]
        so-elasticsearch ------------------ [ OK ]
        so-filebeat ----------------------- [ OK ]
        so-logstash ----------------------- [ OK ]
        so-nginx -------------------------- [ OK ]
        so-redis -------------------------- [ OK ]
        so-sensoroni ---------------------- [ OK ]
        so-steno -------------------------- [ OK ]
        so-strelka-backend ---------------- [ OK ]
        so-strelka-coordinator ------------ [ OK ]
        so-strelka-filestream ------------- [ OK ]
        so-strelka-frontend --------------- [ OK ]
        so-strelka-gatekeeper ------------- [ OK ]
        so-strelka-manager ---------------- [ OK ]
        so-suricata ----------------------- [ OK ]
        so-telegraf ----------------------- [ OK ]
        so-wazuh -------------------------- [ OK ]
        so-zeek --------------------------- [ OK ]
so-raymond_heavynode:

    Checking Docker status

        Docker ---------------------------- [ OK ]

    Checking container statuses

        so-curator ------------------------ [ OK ]
        so-elasticsearch ------------------ [ OK ]
        so-filebeat ----------------------- [ OK ]
        so-logstash ----------------------- [ OK ]
        so-nginx -------------------------- [ OK ]
        so-redis -------------------------- [ OK ]
        so-sensoroni ---------------------- [ OK ]
        so-steno -------------------------- [ OK ]
        so-strelka-backend ---------------- [ OK ]
        so-strelka-coordinator ------------ [ OK ]
        so-strelka-filestream ------------- [ OK ]
        so-strelka-frontend --------------- [ OK ]
        so-strelka-gatekeeper ------------- [ OK ]
        so-strelka-manager ---------------- [ OK ]
        so-suricata ----------------------- [ OK ]
        so-telegraf ----------------------- [ OK ]
        so-wazuh -------------------------- [ OK ]
        so-zeek --------------------------- [ OK ]
so-mc_heavynode:

    Checking Docker status

        Docker ---------------------------- [ OK ]

    Checking container statuses

        so-curator ------------------------ [ OK ]
        so-elasticsearch ------------------ [ OK ]
        so-filebeat ----------------------- [ OK ]
        so-logstash ----------------------- [ OK ]
        so-nginx -------------------------- [ OK ]
        so-redis -------------------------- [ OK ]
        so-sensoroni ---------------------- [ OK ]
        so-steno -------------------------- [ OK ]
        so-strelka-backend ---------------- [ OK ]
        so-strelka-coordinator ------------ [ OK ]
        so-strelka-filestream ------------- [ OK ]
        so-strelka-frontend --------------- [ OK ]
        so-strelka-gatekeeper ------------- [ OK ]
        so-strelka-manager ---------------- [ OK ]
        so-suricata ----------------------- [ OK ]
        so-telegraf ----------------------- [ OK ]
        so-wazuh -------------------------- [ OK ]
        so-zeek --------------------------- [ OK ]
so-chatham_heavynode:

    Checking Docker status

        Docker ---------------------------- [ OK ]

    Checking container statuses

        so-curator ------------------------ [ OK ]
        so-elasticsearch ------------------ [ OK ]
        so-filebeat ----------------------- [ OK ]
        so-logstash ----------------------- [ OK ]
        so-nginx -------------------------- [ OK ]
        so-redis -------------------------- [ OK ]
        so-sensoroni ---------------------- [ OK ]
        so-steno -------------------------- [ OK ]
        so-strelka-backend ---------------- [ OK ]
        so-strelka-coordinator ------------ [ OK ]
        so-strelka-filestream ------------- [ OK ]
        so-strelka-frontend --------------- [ OK ]
        so-strelka-gatekeeper ------------- [ OK ]
        so-strelka-manager ---------------- [ OK ]
        so-suricata ----------------------- [ OK ]
        so-telegraf ----------------------- [ OK ]
        so-wazuh -------------------------- [ OK ]
        so-zeek --------------------------- [ OK ]
so-rochester_heavynode:

    Checking Docker status

        Docker ---------------------------- [ OK ]

    Checking container statuses

        so-curator ------------------------ [ OK ]
        so-elasticsearch ------------------ [ OK ]
        so-filebeat ----------------------- [ OK ]
        so-logstash ----------------------- [ OK ]
        so-nginx -------------------------- [ OK ]
        so-redis -------------------------- [ OK ]
        so-sensoroni ---------------------- [ OK ]
        so-steno -------------------------- [ OK ]
        so-strelka-backend ---------------- [ OK ]
        so-strelka-coordinator ------------ [ OK ]
        so-strelka-filestream ------------- [ OK ]
        so-strelka-frontend --------------- [ OK ]
        so-strelka-gatekeeper ------------- [ OK ]
        so-strelka-manager ---------------- [ OK ]
        so-suricata ----------------------- [ OK ]
        so-telegraf ----------------------- [ OK ]
        so-wazuh -------------------------- [ OK ]
        so-zeek --------------------------- [ OK ]
so-springfield_heavynode:

    Checking Docker status

        Docker ---------------------------- [ OK ]

    Checking container statuses

        so-curator ------------------------ [ OK ]
        so-elasticsearch ------------------ [ OK ]
        so-filebeat ----------------------- [ OK ]
        so-logstash ----------------------- [ OK ]
        so-nginx -------------------------- [ OK ]
        so-redis -------------------------- [ OK ]
        so-sensoroni ---------------------- [ OK ]
        so-steno -------------------------- [ OK ]
        so-strelka-backend ---------------- [ OK ]
        so-strelka-coordinator ------------ [ OK ]
        so-strelka-filestream ------------- [ OK ]
        so-strelka-frontend --------------- [ OK ]
        so-strelka-gatekeeper ------------- [ OK ]
        so-strelka-manager ---------------- [ OK ]
        so-suricata ----------------------- [ OK ]
        so-telegraf ----------------------- [ OK ]
        so-wazuh -------------------------- [ OK ]
        so-zeek --------------------------- [ OK ]
so-carlinville_heavynode:

    Checking Docker status

        Docker ---------------------------- [ OK ]

    Checking container statuses

        so-curator ------------------------ [ OK ]
        so-elasticsearch ------------------ [ OK ]
        so-filebeat ----------------------- [ OK ]
        so-logstash ----------------------- [ OK ]
        so-nginx -------------------------- [ OK ]
        so-redis -------------------------- [ OK ]
        so-sensoroni ---------------------- [ OK ]
        so-steno -------------------------- [ OK ]
        so-strelka-backend ---------------- [ OK ]
        so-strelka-coordinator ------------ [ OK ]
        so-strelka-filestream ------------- [ OK ]
        so-strelka-frontend --------------- [ OK ]
        so-strelka-gatekeeper ------------- [ OK ]
        so-strelka-manager ---------------- [ OK ]
        so-suricata ----------------------- [ OK ]
        so-telegraf ----------------------- [ OK ]
        so-wazuh -------------------------- [ OK ]
        so-zeek --------------------------- [ OK ]
so-main_heavynode:

    Checking Docker status

        Docker ---------------------------- [ OK ]

    Checking container statuses

        so-curator ------------------------ [ OK ]
        so-elasticsearch ------------------ [ OK ]
        so-filebeat ----------------------- [ OK ]
        so-logstash ----------------------- [ OK ]
        so-nginx -------------------------- [ OK ]
        so-redis -------------------------- [ OK ]
        so-sensoroni ---------------------- [ OK ]
        so-steno -------------------------- [ OK ]
        so-strelka-backend ---------------- [ OK ]
        so-strelka-coordinator ------------ [ OK ]
        so-strelka-filestream ------------- [ OK ]
        so-strelka-frontend --------------- [ OK ]
        so-strelka-gatekeeper ------------- [ OK ]
        so-strelka-manager ---------------- [ OK ]
        so-suricata ----------------------- [ OK ]
        so-telegraf ----------------------- [ OK ]
        so-wazuh -------------------------- [ OK ]
        so-zeek --------------------------- [ OK ]
so-manager_managersearch:

    Checking Docker status

        Docker ---------------------------- [ OK ]

    Checking container statuses

        so-aptcacherng -------------------- [ OK ]
        so-cortex ------------------------- [ OK ]
        so-curator ------------------------ [ OK ]
        so-dockerregistry ----------------- [ OK ]
        so-elastalert --------------------- [ OK ]
        so-elasticsearch ------------------ [ OK ]
        so-filebeat ----------------------- [ OK ]
        so-fleet -------------------------- [ OK ]
        so-grafana ------------------------ [ OK ]
        so-idstools ----------------------- [ OK ]
        so-influxdb ----------------------- [ OK ]
        so-kibana ------------------------- [ OK ]
        so-kratos ------------------------- [ OK ]
        so-logstash ----------------------- [ OK ]
        so-mysql -------------------------- [ OK ]
        so-nginx -------------------------- [ OK ]
        so-playbook ----------------------- [ OK ]
        so-redis -------------------------- [ OK ]
        so-sensoroni ---------------------- [ OK ]
        so-soc ---------------------------- [ OK ]
        so-soctopus ----------------------- [ OK ]
        so-telegraf ----------------------- [ OK ]
        so-thehive ------------------------ [ OK ]
        so-thehive-es --------------------- [ OK ]
        so-wazuh -------------------------- [ OK ]

**Comment by TOoSmOotH** [comment]
Give it a few minutes to trickle down to influx. There is a delay of a few minutes.

**Reply by stondino00** [reply]
All nodes have been up for over an hour and still shows the above Grid faults.  Something isn't working right.

**Reply by TOoSmOotH** [reply]
Maybe try a forced refresh

**Reply by stondino00** [reply]
May I ask what you mean by a forced refresh?  I've logged out and back into the grid page and rebooted the manager and nodes.

**Reply by TOoSmOotH** [reply]
ctrl shift refresh but logging out and back in would do it as well. What does your soc log say on the manager? Does your grafana work at all?

**Reply by stondino00** [reply]
Yes Grafana works for each node showing data up until the current time so that shows fine.

**Reply by stondino00** [reply]
Grid tab showed all nodes connected fine on .40 release for what that is worth.

**Reply by TOoSmOotH** [reply]
You will need to look at the soc log then and see why it is unable to talk to influxdb then

**Reply by TOoSmOotH** [reply]
They are connected. The issue is its not reading the sostatus info from influxdb

**Reply by stondino00** [reply]
Where is the soc log located? Thanks

**Reply by TOoSmOotH** [reply]
/opt/so/log/soc

**Reply by stondino00** [reply]
This is all that's in /opt/so/log/soc/sensoroni-server.log


[sohybridad@SO-Manager soc]$ tail -f sensoroni-server.log
timestamp=2021-04-28T23:48:19.404083341Z level=info message="Handled request" contentLength=20 elapsedMs=106 impl=*server.NodeHandler method=POST path=/api/node remoteAddr=172.17.0.1:37200 sourceIp=10.2.6.222 statusCode=200 userId=
timestamp=2021-04-28T23:48:19.405421887Z level=info message="Handled request" contentLength=20 elapsedMs=111 impl=*server.NodeHandler method=POST path=/api/node remoteAddr=172.17.0.1:37196 sourceIp=192.168.20.57 statusCode=200 userId=
timestamp=2021-04-28T23:48:19.431031092Z level=info message="Handled request" contentLength=20 elapsedMs=0 impl=*server.NodeHandler method=POST path=/api/node remoteAddr=172.17.0.1:37204 sourceIp=10.3.6.222 statusCode=200 userId=
timestamp=2021-04-28T23:48:20.42771526Z level=info message="Handled request" contentLength=718 elapsedMs=0 impl=*server.GridHandler method=GET path=/api/grid remoteAddr=172.17.0.1:37216 sourceIp=10.200.93.188 statusCode=200 userId=6bc808bd-ddf0-4540-91ad-e89c0e0f9e20
timestamp=2021-04-28T23:48:20.818408327Z level=info message="Handled request" contentLength=20 elapsedMs=2 impl=*server.NodeHandler method=POST path=/api/node remoteAddr=172.17.0.1:37220 sourceIp=10.11.6.222 statusCode=200 userId=
timestamp=2021-04-28T23:48:21.634947312Z level=info message="Handled request" contentLength=20 elapsedMs=0 impl=*server.NodeHandler method=POST path=/api/node remoteAddr=172.17.0.1:37232 sourceIp=10.15.6.222 statusCode=200 userId=
timestamp=2021-04-28T23:48:21.713274818Z level=info message="Handled request" contentLength=20 elapsedMs=0 impl=*server.NodeHandler method=POST path=/api/node remoteAddr=172.17.0.1:37236 sourceIp=10.4.6.222 statusCode=200 userId=
timestamp=2021-04-28T23:48:23.603059704Z level=info message="Handled request" contentLength=20 elapsedMs=1 impl=*server.NodeHandler method=POST path=/api/node remoteAddr=172.17.0.1:37240 sourceIp=10.8.6.222 statusCode=200 userId=
timestamp=2021-04-28T23:48:24.540083499Z level=info message="WebSocket message received" msg="{ \"Kind\": \"Ping\" }" path=/ws remoteAddr=172.17.0.1:58762 sourceIp=10.200.93.188 type=1
timestamp=2021-04-28T23:48:24.812771235Z level=info message="Handled request" contentLength=20 elapsedMs=0 impl=*server.NodeHandler method=POST path=/api/node remoteAddr=172.17.0.1:37244 sourceIp=192.168.20.59 statusCode=200 userId=
timestamp=2021-04-28T23:48:28.078822735Z level=info message="Handled request" contentLength=20 elapsedMs=0 impl=*server.NodeHandler method=POST path=/api/node remoteAddr=172.17.0.1:37256 sourceIp=10.5.6.222 statusCode=200 userId=
timestamp=2021-04-28T23:48:29.420218549Z level=info message="Handled request" contentLength=20 elapsedMs=1 impl=*server.NodeHandler method=POST path=/api/node remoteAddr=172.17.0.1:37276 sourceIp=192.168.20.57 statusCode=200 userId=
timestamp=2021-04-28T23:48:29.427423345Z level=info message="Handled request" contentLength=20 elapsedMs=0 impl=*server.NodeHandler method=POST path=/api/node remoteAddr=172.17.0.1:37280 sourceIp=10.11.6.222 statusCode=200 userId=
timestamp=2021-04-28T23:48:29.439427659Z level=info message="Handled request" contentLength=20 elapsedMs=0 impl=*server.NodeHandler method=POST path=/api/node remoteAddr=172.17.0.1:37284 sourceIp=10.11.6.222 statusCode=200 userId=
timestamp=2021-04-28T23:48:30.858372706Z level=info message="Handled request" contentLength=20 elapsedMs=2 impl=*server.NodeHandler method=POST path=/api/node remoteAddr=172.17.0.1:37308 sourceIp=10.11.6.222 statusCode=200 userId=
timestamp=2021-04-28T23:48:31.663803078Z level=info message="Handled request" contentLength=20 elapsedMs=0 impl=*server.NodeHandler method=POST path=/api/node remoteAddr=172.17.0.1:37322 sourceIp=10.12.6.222 statusCode=200 userId=
timestamp=2021-04-28T23:48:31.728104927Z level=info message="Handled request" contentLength=20 elapsedMs=0 impl=*server.NodeHandler method=POST path=/api/node remoteAddr=172.17.0.1:37326 sourceIp=10.13.6.222 statusCode=200 userId=
timestamp=2021-04-28T23:48:33.633214667Z level=info message="Handled request" contentLength=20 elapsedMs=1 impl=*server.NodeHandler method=POST path=/api/node remoteAddr=172.17.0.1:37332 sourceIp=10.14.6.222 statusCode=200 userId=
timestamp=2021-04-28T23:48:34.095178434Z level=info message="WebSocket message received" msg="{ \"Kind\": \"Ping\" }" path=/ws remoteAddr=172.17.0.1:37188 sourceIp=10.200.93.188 type=1
timestamp=2021-04-28T23:48:35.119390793Z level=info message="Handled request" contentLength=20 elapsedMs=0 impl=*server.NodeHandler method=POST path=/api/node remoteAddr=172.17.0.1:37336 sourceIp=192.168.20.59 statusCode=200 userId=

**Reply by stondino00** [reply]
Not sure if it's related but Kibana isn't acting right after this .50  upgrade.  I've never had issues like this in the previous upgrade versions but Kibana is complaining about not being able to find a license and giving errors for the remote cluster nodes and won't display data when I open it.  Things are pretty broken for me after doing the .50 upgrade.

**Reply by TOoSmOotH** [reply]
On the manager run the following:  

sudo salt \* cmd.run "cat /opt/so/log/sostatus/sostatus.log"

Also run on the manager: 

sudo ls -al /opt/so/saltstack/local/salt/telegraf/etc

**Reply by TOoSmOotH** [reply]
For the Kibana thing please start a new thread

**Reply by stondino00** [reply]
sudo salt * cmd.run "cat /opt/so/log/sostatus/sostatus.log"
No minions matched the target. No command was sent, no jid was assigned.
ERROR: No return received


sudo ls -al /opt/so/saltstack/local/salt/telegraf/etc
total 0
drwxr-xr-x. 2 socore socore  6 Dec  9 21:00 .
drwxr-xr-x. 4 socore socore 32 Dec  9 21:00 ..

**Reply by TOoSmOotH** [reply]
you need a \ before the *

**Reply by stondino00** [reply]
[sohybridad@SO-Manager ~]$ sudo salt \* cmd.run "cat /opt/so/log/sostatus/sostatus.log"
[sudo] password for sohybridad:
so-raymond_heavynode:
    cat: /opt/so/log/sostatus/sostatus.log: No such file or directory
so-mc_heavynode:
    cat: /opt/so/log/sostatus/sostatus.log: No such file or directory
so-farmersville_heavynode:
    cat: /opt/so/log/sostatus/sostatus.log: No such file or directory
so-manager_managersearch:
    cat: /opt/so/log/sostatus/sostatus.log: No such file or directory
so-chatham_heavynode:
    cat: /opt/so/log/sostatus/sostatus.log: No such file or directory
so-rochester_heavynode:
    cat: /opt/so/log/sostatus/sostatus.log: No such file or directory
so-carlinville_heavynode:
    cat: /opt/so/log/sostatus/sostatus.log: No such file or directory
so-main_heavynode:
    cat: /opt/so/log/sostatus/sostatus.log: No such file or directory
so-springfield_heavynode:
    cat: /opt/so/log/sostatus/sostatus.log: No such file or directory
ERROR: Minions returned with non-zero exit code

**Reply by stondino00** [reply]
There is a status.log file but no sostatus.log.


sudo salt \* cmd.run "cat /opt/so/log/sostatus/status.log"
[sudo] password for sohybridad:
so-farmersville_heavynode:
    0
so-raymond_heavynode:
    0
so-mc_heavynode:
    0
so-manager_managersearch:
    0
so-carlinville_heavynode:
    0
so-chatham_heavynode:
    0
so-rochester_heavynode:
    0
so-springfield_heavynode:
    0
so-main_heavynode:
    0
[sohybridad@SO-Manager sostatus]$

**Reply by TOoSmOotH** [reply]
try    

curl -skG 'https://localhost:8086/query?db=telegraf' --data-urlencode 'q=select * from sostatus where time > now() - 5m'

**Reply by stondino00** [reply]
curl -skG 'https://localhost:8086/query?db=telegraf' --data-urlencode 'q=select * from sostatus where time > now() - 5m'
{"results":[{"statement_id":0,"series":[{"name":"sostatus","columns":["time","host","status"],"values":[["2021-04-29T15:17:00Z","so-chatham",0],["2021-04-29T15:17:00Z","so-farmersville",0],["2021-04-29T15:17:00Z","so-main",0],["2021-04-29T15:17:00Z","so-mc",0],["2021-04-29T15:17:00Z","so-rochester",0],["2021-04-29T15:17:00Z","so-springfield",0],["2021-04-29T15:18:00Z","so-chatham",0],["2021-04-29T15:18:00Z","so-farmersville",0],["2021-04-29T15:18:00Z","so-manager",0],["2021-04-29T15:18:00Z","so-mc",0],["2021-04-29T15:18:00Z","so-springfield",0],["2021-04-29T15:19:00Z","so-carlinville",0],["2021-04-29T15:19:00Z","so-chatham",0],["2021-04-29T15:19:00Z","so-manager",0],["2021-04-29T15:19:00Z","so-raymond",0],["2021-04-29T15:19:00Z","so-rochester",0],["2021-04-29T15:20:00Z","so-carlinville",0],["2021-04-29T15:20:00Z","so-farmersville",0],["2021-04-29T15:20:00Z","so-raymond",0],["2021-04-29T15:21:00Z","so-carlinville",0],["2021-04-29T15:21:00Z","so-chatham",0],["2021-04-29T15:21:00Z","so-farmersville",0],["2021-04-29T15:21:00Z","so-manager",0],["2021-04-29T15:21:00Z","so-mc",0],["2021-04-29T15:21:00Z","so-raymond",0],["2021-04-29T15:21:00Z","so-springfield",0]]}]}]}

**Comment by Kennet-B** [comment]
i have same problem
![Screenshot_2021-04-29  Security Onion](https://user-images.githubusercontent.com/74116181/116556670-02f98a80-a8fe-11eb-88af-6e5bd00af002.png)

**Reply by TOoSmOotH** [reply]
is so-status green on the ones that show fault?

**Comment by ow111** [comment]
I am also seeing this issue - Although only on a forward node. From what i can tell still seeing traffic and zeek logs and all services are ok.. (so-status)

**Comment by jertel** [comment, marked as answer]
The problem is casing of the hostnames. By default, SOC assumes the node's ID is the hostname. It appears some of your grids are using mixed case hostnames. Yet in InfluxDB the host tag is always lowercase. We'll get that improved in the next release.

In the meantime, you can execute the following commands on your manager node to pull in this lowercase update:

```
sudo curl https://raw.githubusercontent.com/Security-Onion-Solutions/securityonion/dev/salt/sensoroni/files/sensoroni.json -o /opt/so/saltstack/default/salt/sensoroni/files/sensoroni.json

sudo salt "*" state.apply sensoroni

sudo so-soc-restart
```

If you take this approach we'd like to hear if it resolved the issue for you.

**Reply by stondino00** [reply]
Ran that with 0 under failed and so far same issue for me in GRID with my mixed casing names showing faults.  I'm also showing issues in Kibana with remote clusters that I will create another discussion for.  Not sure if this is all related.

**Reply by ow111** [reply]
This fixed the issue for me, Thanks @jertel

**Reply by jertel** [reply]
@stondino00 Does your grid now show lowercase IDs on the left-most column?

**Reply by stondino00** [reply]
Correction that did fix the grid issues.  Showing all green now.  Thanks for looking into this.

**Reply by n0th1nt0lOOz** [reply]
Hi all,
After running cmd above, nothing change. Do we have any other way? Any log I need to provide?
Thanks
![image](https://user-images.githubusercontent.com/28757980/116976599-96123600-aceb-11eb-910b-eb74ab742a89.png)

**Reply by jertel** [reply]
You blurred out the IDs which makes it difficult to assist you. Are your IDs all lower case? If not then the fix was not successfully applied. If the IDs on the left column are all lowercase then there's a different issue causing your faults, in which case you will need to open a separate discussion and provide more information such as logs, and then also expanding the grid nodes rows so we can see what is causing the fault.

**Reply by n0th1nt0lOOz** [reply]
@jertel sorry for that. All IDs are in lowercase. When expanding the status I have this one:
![image](https://user-images.githubusercontent.com/28757980/117097678-75eb8100-ad96-11eb-8917-afadccd6b1a5.png)
I don't see any ticket/issue relate to this problem on SO git discussion yet

**Comment by stondino00** [comment]
Would this cause an issue with the alerts tab?

I'm showing a constant 500 error and no alerts now after performing this fix.  I randomly would get it before but now I get nothing for alerts and simply a 500 error.

![image](https://user-images.githubusercontent.com/54154986/116593050-4a741c80-a8e6-11eb-9ece-81059d8e4699.png)

**Comment by jcurtiss7492** [comment]
Any luck on a solution? I'm in the same boat.

**Reply by TOoSmOotH** [reply]
Did you apply the hotfix?

**Reply by n0th1nt0lOOz** [reply]
I applied the hotfix and my hostnames are all lowercase but nothing change

**Reply by jcurtiss7492** [reply]
@TOoSmOotH Yes, I was able to get my hostname to all lowercase by using the commands you supplied.  It changed the status on Grid to "OK", but it still says, "Appliance images unavailable" when you expand to show node details.

**Reply by TOoSmOotH** [reply]
@tientmse62290 You will need to look at your sensoroni logs on your sensor then.

@jcurtiss7492 Those only display images for SO appliances.

**Reply by jcurtiss7492** [reply]
@TOoSmOotH 10-4, thanks!  I'll check my sensoroni logs also getting random "Request failed with status code 500" messages on Alerts screen, especially when I try to "Drilldown" on an alert.

**Reply by n0th1nt0lOOz** [reply]
@TOoSmOotH Hi, my sensoroni show no error and I still got the problem
![image](https://user-images.githubusercontent.com/28757980/117603187-ac514380-b17c-11eb-8c4c-4ddf15274e26.png)

**Reply by xme** [reply]
Jumping into the thread... Same for me. all is green based on so-status, names are lowercase (but contain "-" like "so-master"), and impossible to get rid of the "Fault" red icon :( Also, EPS remains at "0" while alerts, pcaps, etc... are working smoothly.

**Reply by jertel** [reply]
Run the following command on your manager node and let us know what you get back:
```
sudo docker exec  so-influxdb influx -ssl -unsafeSsl -database telegraf -execute 'select * from sostatus; select * from fbstats'
```

**Reply by xme** [reply]
Uh Oh.... I don't have a so-influxdb container!?!

```
[root@so-manager ~]# so-status

Checking Docker status

    Docker ------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]

Checking container statuses

    so-aptcacherng ----------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-curator --------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-dockerregistry -------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-elastalert ------------------------------------------------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-elasticsearch --------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-filebeat -------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-idstools -------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-kibana ---------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-kratos ---------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-logstash -------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-nginx ----------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-redis ----------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-sensoroni ------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-soc ------------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-soctopus -------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-telegraf -------------------------------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
```

**Reply by jertel** [reply]
Those still encountering this issue -- did you all opt-out of installing Grafana during setup?

**Reply by xme** [reply]
Yes (in my case)

**Reply by n0th1nt0lOOz** [reply]
Mine is: 
#sudo docker exec  so-influxdb influx -ssl -unsafeSsl -database telegraf -execute 'select * from sostatus; select * from fbstats'
Error: No such container: so-influxdb

so-status

Checking Docker status

    Docker ---------------------------------------------------------------------------------------------------------------------------------------- [ OK ]

Checking container statuses

    so-aptcacherng -------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-cortex ------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-curator ------------------------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-dockerregistry ----------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-elastalert --------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-elasticsearch ------------------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-filebeat ----------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-fleet -------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-idstools ----------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-kibana ------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-kratos ------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-logstash ----------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-mysql -------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-nginx -------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-playbook ----------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-redis -------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-sensoroni ---------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-soc ---------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-soctopus ----------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-telegraf ----------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-thehive ------------------------------------------------------------------------------------------------------------------------------------ [ OK ]
    so-thehive-es --------------------------------------------------------------------------------------------------------------------------------- [ OK ]
    so-wazuh -------------------------------------------------------------------------------------------------------------------------------------- [ OK ]
Same as @xme

**Reply by jertel** [reply]
Thanks for the confirmation. Security Onion will need to account for this setup scenario with a new release.

**Reply by jertel** [reply]
Tracking this issue in #4207.

**Reply by xme** [reply]
Tx! In the mean time, any manual operations to perform to install the missing component manually?

**Reply by jertel** [reply]
This was fixed in version 2.3.51.
